patternfly/patternfly-elements
# fix: saturated visited link color

## Component name

- all components


### Related issue

- https://github.com/patternfly/patternfly-elements/issues/1126


### Preview



https://deploy-preview-1127--happy-galileo-ea79c4.netlify.app/elements/pfe-card/demo/


### What has changed and why
 
- Update visited link color to something that would pass accessibility standards on saturated backgrounds

### Testing instructions

1. Go to pfe-card demo
2. Click on a link in the complement or accent card so that its "visited"
3. Notice the link color is no longer purple, but a light gray instead.
 

### Ready-for-merge Checklist

Check off items as they are completed.  Feel free to delete items if they are not applicable.



### Merging

Please **squash** when merging and ensure your commit message uses [conventional commit](https://www.conventionalcommits.org/en/v1.0.0/#summary) formatting.
